src/buttons/button-checkbox.directive.ts

```diff
@@ -26,11 +26,13 @@ export const CHECKBOX_CONTROL_VALUE_ACCESSOR: any = {
 })
 export class ButtonCheckboxDirective implements ControlValueAccessor, OnInit {
   /** Truthy value, will be set to ngModel */
-  @Input() btnCheckboxTrue: any = true;
+  @Input() btnCheckboxTrue = true;
```

Comment:
`any` must be there because we need the ability to set other values as possible values of the button.